prusti-tests/tests/verify/pass/floats/predicate_float.rs

```diff
@@ -0,0 +1,14 @@
+extern crate prusti_contracts;
+use prusti_contracts::*;
+
+#[predicate]
```

Comment:
This is old syntax (https://github.com/viperproject/prusti-dev/pull/537). Also what is this testing? These functions can simply be `#[pure]`, they are not using Prusti syntax.